tiled/server/router.py

```diff
@@ -1215,3 +1219,104 @@ async def delete_revision(
 
     await entry.delete_revision(number)
     return json_or_msgpack(request, None)
+
+
+@router.get("/asset/bytes/{path:path}")
+async def get_asset(
+    request: Request,
+    id: int,
+    relative_path: Optional[Path] = None,
```

Comment:
I believe fastAPI got rid of optional path parameters in recent versions

Reply:
I could be wrong, but I think this comment is conflating:

- URL path parameters like (`{path:path}`) above, which is consumed by the `entry` dependency below
- this `relative_path` _query_ parameter, given as `?relative_path=...`, which happens to be typed as `Path`

If query parameters of type `Path` cannot be optional, can you link to the docs/changelog?